responders/src/UI/embc-responder/src/app/core/services/config.service.ts

```diff
@@ -23,35 +23,35 @@ export class ConfigService {
     private cacheService: CacheService
   ) {}
 
-  public load(): Observable<Configuration> {
-    if (this.config != null) {
-      return of(this.config);
-    }
-    return this.configurationService.configurationGetConfiguration().pipe(
-      tap((c) => {
-        this.config = { ...c };
-      })
-    );
-  }
-
-  public async getAuthConfig(): Promise<AuthConfig> {
-    return await this.load()
+  public async load(): Promise<Configuration> {
+    return this.configurationService
```

Comment:
we need to return from service if this.config is not null (same as earlier, returns promise instead of observable)

Reply:
Is this the way it should look like now?